buildSrc/src/main/groovy/io/micronaut/gradle/MicronautLogosTask.groovy

```diff
@@ -0,0 +1,34 @@
+package io.micronaut.gradle
+
+import groovy.transform.CompileStatic
+import io.micronaut.gallery.MicronautLogosPage
+import org.gradle.api.DefaultTask
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.api.tasks.TaskAction
+
+@CompileStatic
+class MicronautLogosTask extends DefaultTask {
+
+    static final String PAGE_NAME_MN_LOGOS = "micronaut-logos.html"
+
+    @Input
+    final Property<File> logos = project.objects.property(File)
+
+    @OutputDirectory
+    final Property<File> output = project.objects.property(File)
+
+    @TaskAction
+    void renderDocsPage() {
```

Comment:
wrong method name. It should be probably named `renderLogosPage`

Reply:
Good call. I'll update the `QuestsionsTask` too.